bstrp_functions.sh

```diff
@@ -75,30 +75,31 @@ set_incompatible_with(){
 }
 
 print_multi_option_status(){
-  feature="$1"
   feature_status=${!1}
   declare -a VAR_OPTS=()
 
-  declare VAR_OPTS=$1_OPTIONS[@]
-  VAR_OPTS=$1_OPTIONS[@]
+  declare VAR_OPTS=("$1_OPTIONS[@]")
+  VAR_OPTS=("$1_OPTIONS[@]")
 
   for option in "${!VAR_OPTS}" ; do
     if [ "${option}" = "$feature_status" ]; then
+      # shellcheck disable=SC2059
     	printf "${RED}"
     fi
+    # shellcheck disable=SC2059
     printf "${option}"
+    # shellcheck disable=SC2059
     printf "${NO_COLOR} "
```

Comment:
This could be rewritten as
```suggestion
    if [ "${option}" = "$feature_status" ]; then
    	printf "%b%s%b " "${RED}" "${option}" "${NO_COLOR}"
    else
    	printf "%s " "${option}"
    fi
```
which I think would be more readable.

Reply:
Great suggestion! Updated in 4650e8b54fcab4ef1c6a709a2cfdc21eadb2903b